docs/pages/Configuration.md

```diff
@@ -303,6 +317,20 @@ builder:
         color: blue
 ````
 
+### JSDoc
+You can exclude a projects resources from the JSDoc build process using a list of glob patterns. Matching resources will be ignored by the JSDoc build task.
+
+Patterns are always applied relative to the projects virtual **source** directory `/resources/`.
+
+Any general builder excludes (as defined in `builder.resources.excludes`) are applied *after* these excludes.
```

Comment:
These excludes are applied *before* any general builder excludes that have been defined in `builder.resources.excludes`.

Reply:
See https://github.com/SAP/ui5-tooling/pull/258